src/qa-certification-event-workspace/qa-certification-event-workspace.service.ts

```diff
@@ -231,7 +236,7 @@ export class QACertificationEventWorkspaceService {
     entity.evalStatusCode = 'EVAL';
     entity.submissionAvailabilityCode = 'REQUIRE';
 
-    await this.repository.save(entity);
+    await repository.save(entity);
 
     return this.getQACertEvent(entity.id);
```

Comment:
This should be updated to accept the transaction as a parameter (and the method should be updated to use it):
```suggestion
    return this.getQACertEvent(entity.id, trx);
```

Reply:
updated.